samples/Playground/Program.cs

```diff
@@ -27,8 +29,13 @@ public static async Task<int> Main(string[] args)
             // To attach to the children
             Microsoft.Testing.TestInfrastructure.DebuggerUtility.AttachCurrentProcessToParentVSProcess();
             ITestApplicationBuilder testApplicationBuilder = await TestApplication.CreateBuilderAsync(args);
+
+            // Test MSTest
             testApplicationBuilder.AddMSTest(() => [Assembly.GetEntryAssembly()!]);
 
+            // Test a custom local test framework
+            // testApplicationBuilder.RegisterTestFramework(_ => new TestFrameworkCapabilities(), (_, _) => new DummyAdapter());
```

Comment:
Should this be commented out, or uncommented?

Reply:
I decided to keep the playground in a state where MSTest is default while keeping the code for a custom adapter so that we don't have to always recreate the various lines. Ideally, I would need to spend a little time to use a variable/const or some pragma to switch between MSTest and custom adapter but that's not urgent as far of this PR is concerned.